script.js

```diff
@@ -0,0 +1,81 @@
+// hamburger
+
+const ham = document.querySelector('.ham');
+const logo = document.querySelector('.logo');
+const icons = document.querySelector('.icons');
+const iconLink = document.querySelector('.icon-link');
+
+ham.addEventListener('click', () => {
+  if (ham.classList.contains('active')) {
+    ham.classList.remove('active');
+    icons.style.display = 'none';
+  } else {
+    ham.classList.add('active');
+    icons.style.display = 'block';
+  }
+});
+
+iconLink.addEventListener('click', () => {
+  ham.classList.toggle('active');
+  icons.style.display = 'none';
+  logo.style.visibility = 'visible';
+});
+
+// dynamic page
+
+const speakerSection = document.querySelector('#featuredM');
+
+const desk = [{
+  name: 'Mc Richie',
+  description: 'Summer festival Concert hosted by Mc Richie',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+
+}, {
+  name: 'DJ Maphorisa',
+  description: 'South african based musician.',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}, {
+  name: 'Rihanna',
+  description: 'New York based musician.',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}, {
+  name: 'Larmack',
+  description: 'Jamaica based musician.',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}, {
+  name: 'Dladla mshunqisi',
+  description: 'KZN based musician.',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}, {
+  name: 'Tatiana',
+  description: 'Kenya based musician.',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}];
```

Comment:
<img width="971" alt="Screen Shot 2022-06-30 at 9 04 23" src="https://user-images.githubusercontent.com/82426682/176697650-9512784d-4526-4b5a-af32-39a5ea7e7c46.png">

Please make sure your speaker objects hold all of the information above.